lib/core/api.js

```diff
@@ -166,8 +166,21 @@ api.user.updateProfile = function(userId, updates, cb){
     if(err){
       return cb(err);
     }
+    var currentEmail = _.get(profile, 'emails[0]');
```

Comment:
Yeah, `_.get` is always safe, apparently. I had to check it's behavior in `console` to be sure myself before feeling good about this line.